internal/security/bootstrapper/command/setupacl/command.go

```diff
@@ -57,6 +58,9 @@ const (
 	defaultRetryTimeout        = 30 * time.Second
 	emptyLeader                = `""`
 	emptyToken                 = ""
+
+	// environment variable contains a comma separated list of registry role names to be added
+	addRegistryRolesEnvKey = "ADD_REGISTRY_ACL_ROLES"
```

Comment:
Please add edgex-doc issue to document this feature.

Reply:
ok, i'll create one issue

Reply:
https://github.com/edgexfoundry/edgex-docs/issues/367